src/online_providers/stripe_sca.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  handleOnlinePayment: ({ settings, payment }) => {
+    const token = settings.token;
+    const clientSecret = payment.clientSecret;
+
```

Comment:
Redundant whitespace?

Reply:
It have a meaning to separate logical parts of method - prepare/process input, act, return result. This is part of preparing data to be used in method, i.e its dependencies.